### DISCUSSION TITLE
Standalone NodeGraph Component

### DISCUSSION THREAD

**Discussion by bombomby** [Jcparkyn/nodexr#121] [Ideas]
Hi, thanks for putting together a nice project. 
I've been looking for a NodeGraph component in Blazor for quite a while and your implementation seems to tick all the boxes. 
It would be great to see the code for NodeGraph being extracted into its own generic NodeGraph component (according to the comments there was already an intention to do this).
I am pretty sure that there will be a lot of projects (especially in the GameDev industry) that could benefit from such control.

**Comment by MSKRoss** [comment]
I wanted to follow-up on this since I also have a use-case for a NodeGraph UI component in Blazor, but not for Regular Expressions per se. Can this component currently be used in this way (perhaps with some specific overrides)? @Jcparkyn

**Comment by Jcparkyn** [comment]
@MSKRoss This has been requested a couple of times before, and the answer is essentially that it's possible, but it won't be easy. You can see my responses to similar questions [here](https://github.com/Jcparkyn/nodexr/issues/122#issuecomment-1162541456) and [here](https://github.com/Jcparkyn/nodexr/issues/117#issuecomment-877730869) for some more details.
